app/src/main/java/ch/epfl/sdp/lobby/game/IGameLobbyRepository.kt

```diff
@@ -0,0 +1,22 @@
+package ch.epfl.sdp.lobby.game
+
+import ch.epfl.sdp.game.data.Participation
+import ch.epfl.sdp.lobby.PlayerParametersFragment
+
+interface IGameLobbyRepository {
+
+
+    fun getGameId(cb : (Int) -> Unit)
```

Comment:
Use the callback type defined in db instead 
For example:
```suggestion
    fun getGameId(cb: Callback<Int>)
```